src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -134,8 +146,10 @@ private Map<String,String> getOrderTemplateAndFile(CreateSelectOrderOptionsEnum
                 fieldsMap.put(PrlAppsConstants.GENERATE_FILE_NAME, c47aFile);
                 break;
             case generalForm:
-                fieldsMap.put(PrlAppsConstants.TEMPLATE,"");
-                fieldsMap.put(PrlAppsConstants.FILE_NAME, "");
+                fieldsMap.put(PrlAppsConstants.TEMPLATE,n117DraftTemplate);
```

Comment:
Can we please rename these 4 constants to more meaningful names like 'DRAFT_TEMPLATE_NAME', 'DRAFT_FILE_NAME', 'FINAL_TEMPLATE_NAME', 'FINAL_FILE_NAME'